data/changelog.txt

```diff
@@ -1240,6 +1260,13 @@ Audit of Wood panel from Beam and Sheet to replace Byproduct with Result Multi
 Fix 0.5 L I1 Diesel engine not consuming fuel
 'None'-skill crafting counts as max level, not level 0
 [Sky Island] Fix possible painkiller death loop
+Suppressors noise reduction multiplicative instead of additive
+Fix location Variables targeting specific map features
+[Sky Island] Remove incorporeal immunity in death EoC
+Set owner faction of in-progress crafts
+[MoM] Fix Electron Overflow divide by zero
+Fix crash after deleting items in character_takeoff_item eoc
+Mi-go pathfinding avoids fire
```

Comment:
```suggestion
Suppressors noise reduction multiplicative instead of additive
Fix location Variables targeting specific map features
Set owner faction of in-progress crafts
[MoM] Fix Electron Overflow divide by zero
Fix crash after deleting items in character_takeoff_item eoc
Mi-go pathfinding avoids fire
```